### DISCUSSION TITLE
Runtimes Impracticality

### DISCUSSION THREAD

**Discussion by aervxa** [d2phap/ImageGlass#1652] [Polls]
I want to run ImageGlass as I feel like it's one of the best 3r-party Image-Viewing Applications. But then I am required to install .NET Runtime and WebView2 to even run it.
Why can't it all be bundled into one application?

**Comment by d2phap** [comment]
It's to save the disk. Once you install the .NET Runtime, you can use it for other apps. Similar to Webview2 runtime, I believe, most of Windows 10, 11 already installed them.
